src/commands/deploy-contract.ts

```diff
@@ -148,31 +151,32 @@ only the raw transaction hex will be logged.
     }
 
     if (flags.broadcast) {
-      const result = await broadcastTransaction(tx, network);
-      if (typeof result === 'string') {
+      try {
+        const { txid: result } = await broadcastTransaction({
+          transaction: tx,
+          network,
+        });
         if (verbose) {
           this.log('Transaction ID:', result);
           const explorerLink = `https://explorer.stacks.co/txid/0x${result}`;
-          !(network instanceof StacksMocknet) &&
+          network.magicBytes !== STACKS_MOCKNET.magicBytes &&
```

Comment:
This is likely different behavior. Mocknet magicbytes are the same as Testnet

Reply:
Is there a good way to check if the given network is actually testnet|mainnet? Otherwise we'll just remove this check.

Reply:
I removed these checks. The explorer link will always be shown https://github.com/stacks-network/send-many-stx-cli/pull/16/commits/a32cc0450603acba09d18dc0133ed901a968d362

The devnet/mocknet use-case can be dealt with later if it becomes a thing.

Reply:
Only with the default url, but that's not very clean either

Reply:
Ah yeah good idea, done